zgw_consumers/admin_fields.py

```diff
@@ -35,11 +35,11 @@ def get_zaaktypen() -> Dict[Service, List[Dict[str, Any]]]:
         while response["next"]:
             next_url = urlparse(response["next"])
             query = parse_qs(next_url.query)
-            new_page = int(query["page"][0]) + 1
+            new_page = int(query["page"][0])
```

Comment:
Had to check if our new pagination helper is also affected, but that one doesn't bother with parsing stuff at all so nice one :tada: 

I had a suspicion it had something to do with pagination, I just expected it to end up in an infinite loop :thinking: